docassemble/EFSPIntegration/efm_client.py

```diff
@@ -527,15 +530,15 @@ def serialize_person(person:Union[Person,Individual])->Dict:
     }
   else:
     return_dict = {
-      "firmName": person.name.text if hasattr(person.name, 'text') else None,
+      "firmName": person.name.text,
     }
   return_dict.update({
     "address": {
-      "addressLine1": person.address.address if hasattr(person.address, 'address') else None,
+      "addressLine1": person.address.address,
       "addressLine2": person.address.unit if hasattr(person.address, 'unit') else None,
-      "city": person.address.city if hasattr(person.address, 'city') else None,
```

Comment:
Same with these as well, it looks like a merge issue (or just a Docassemble github issue)? There are time where we don't need to provide the address. We can still always ask in the yml, but it shouldn't be forced here.

Reply:
I'll fix this one too